v1.0/Workflow.yml

```diff
@@ -66,6 +66,20 @@ $graph:
           ```
         * The common field `description` has been renamed to `doc`.
 
+      ## Errata
+
+      Post v1.0 release changes to the spec.
+
+        * 12 March 2017:
+          * Mark `default` as not required for link checking.
+          * Add note that recursive subworkflows is not allowed.
+          * Fix mistake in discussion of extracting field names from workflow step ids.
+        * 21 July 2017:
+          * Add clarification about scattering over empty arrays.
+          * Clarify interpretation of secondaryFiles on inputs.
+        * 22 July 2017: (v1.0.1)
+          * Expanded discussion of semantics of File and Directory types
+
```

Comment:
other non mentioned changes
- EMACScript -> E**CMA**Script
- Clarified processing of input object's input parameters when a value is not provided
- Clarified valid types and meaning of the format field on inputs versus outputs
- in the context of a `valueFrom` Expression: Input parameter defaults must be applied before evaluating the
expression